SDWebImage/SDWebImageDownloader.m

```diff
@@ -271,8 +296,9 @@ - (nullable SDWebImageDownloadToken *)addProgressCallback:(SDWebImageDownloaderP
                                            createCallback:(SDWebImageDownloaderOperation *(^)(void))createCallback {
     // The URL will be used as the key to the callbacks dictionary so it cannot be nil. If it is nil immediately call the completed block with no image or data.
     if (url == nil) {
-        if (completedBlock != nil) {
```

Comment:
We should provide a error, or that may not work as a consistent component with `SDWebImageManager`.